src/components/Slides/SlideViewer.js

```diff
@@ -26,31 +72,24 @@ class SlideViewer extends Component {
 			nextButton: 'next',
 			previousButton: 'previous',
 			showNavigator: true,
-			navigatorAutoFade:  false,
-			navigatorId:   'osd-navigator',
-			tileSources: 'deepZoomImages/' + slideId + '.dzi'
+			navigatorAutoFade: false,
+			navigatorId: 'osd-navigator',
+			tileSources: 'deepZoomImages/' + slideId + '.dzi',
+			overlays: this.state.gridOverlay
 		});
+		this.setState({ viewer: this.viewer });
 	}
 
-	componentDidMount(){
```

Comment:
I am not sure you can/should just get rid of this section of code.

Reply:
Looks like just didUpdate was removed...please restore.

Reply:
Re-added this. Good catch, you are correct, we need this for the slide changing. Haven't been toggling between slides when doing this.